pkg/filesystem/local_directory_test.go

```diff
@@ -234,7 +244,7 @@ func TestLocalDirectoryReadlinkSuccess(t *testing.T) {
 	require.NoError(t, d.Symlink("/foo/bar/baz", path.MustNewComponent("symlink")))
 	target, err := d.Readlink(path.MustNewComponent("symlink"))
 	require.NoError(t, err)
-	require.Equal(t, "/foo/bar/baz", target)
+	require.Equal(t, filepath.FromSlash("/foo/bar/baz"), target)
```

Comment:
I guess it should be the responsibility of LocalDirectory on Windows to call filepath.ToSlash() here. The reason being that the Remote Execution protocol requires us to capture symlink targets using / as a path separator:

https://github.com/bazelbuild/remote-apis/blob/main/build/bazel/remote/execution/v2/remote_execution.proto#L1233-L1238